source/Textures.js

```diff
@@ -0,0 +1,20 @@
+var Pixi = require("pixi.js")
+
+import Junkership from "./Junkership.js"
+
+exports.initTex = function initTex() {
+    PIXI.loader
+        .add("redJunkership",    require("./images/red-starship.png"))
+        .add("yellowJunkership", require("./images/yellow-starship.png"))
+        .add("greenJunkership",  require("./images/green-starship.png"))
+        .add("blueJunkership",   require("./images/blue-starship.png"))
+        .add("projectile",       require("./images/projectile.png"))
+        .add("trashbot",         require("./images/enemy-starship.png"))
+        .load()
+
+        .on("complete",setup)
+}
```

Comment:
Centralizing is always good, IMO. Easier to update filenames and structure if we need to, for one.

Another is the ability to run middleware before or after each module (mostly useful for spritesheets, but I've also been led to believe that we can use that to apply a "rage mode red" tint to the Trashbots, for instance.) Also gives us some built in error handling, progress checks, etc.

Basically, just a more robust utility in general.